PULL_REQUEST_TEMPLATE

```diff
@@ -1,7 +1,22 @@
-[MOB-####](https://ecosia.atlassian.net/browse/MOB-####)
+<!--
+Don't forget to add a prefix to the PR TITLE in this format
+[MOB-####] PR TITLE
+-->
+
+[MOB-####]
 
 ## Context
 
 ## Approach
 
-## Other
+## Other
+
+## Before merging
+
+### Checklist
+
+- [ ] I performed some relevant testing on a real device and/or simulator
+- [ ] I wrote Unit Tests that confirm the expected behaviour
+- [ ] I updated the Localization if needed
```

Comment:
Would it make sense to mention `(only the english source file since Transifex automation does the rest)`?

Reply:
Awesome addition that provides even more clarity!